content/docs/guides/components/trtinferenceserver.md

```diff
@@ -0,0 +1,246 @@
++++
+title = "NVIDIA TensorRT Inference Server"
+description = ""
+weight = 10 
+toc = true
+bref= "Model serving using TRT Inference Server"
+[menu]
+[menu.docs]
+parent = "components"
+weight = 50
++++
+
+# NVIDIA TensorRT Inference Server
+
+NVIDIA TensorRT Inference Server is a REST and GRPC service for deep-learning
+inferencing of TensorRT, TensorFlow and Caffe2 models. The server is
+optimized deploy machine and deep learning algorithms on both GPUs and
+CPUs at scale.
+
+These instructions detail how to set up a GKE cluster suitable for
+running the NVIDIA TensorRT Inference Server and how to use the
+`io.ksonnet.pkg.nvidia-inference-server` prototype to generate
+Kubernetes YAML and deploy to that cluster.
+
+For more information on the NVIDIA TensorRT Inference Server see the [NVIDIA TensorRT
+Inference Server User
+Guide](https://docs.nvidia.com/deeplearning/sdk/inference-user-guide/index.html)
+and the [NVIDIA TensorRT Inference Server
+Clients](https://github.com/NVIDIA/dl-inference-server) open-source
+repository.
+
+## Setup
```

Comment:
The Setup section along with creating cluster, enabling CUDA on GPUs etc. for Google Kubernetes Engine cluster are redundant given that this site already has the latest instructions on how to go about it. Why not point to https://www.kubeflow.org/docs/started/getting-started-gke/ for those instructions. This will setup the Kubeflow deployment on GKE which should also create GPU Node pool that autoscales. Additionally, it will also setup CUDA drivers on any nodes that get allocated to the GPU Pool.

Reply:
Essentially, everything upto line 116 could be removed.